src/pages/constitution/Constitution.tsx

```diff
@@ -58,9 +97,35 @@ class Constitution extends React.Component<Props> {
           </IonToolbar>
         </IonHeader>
         <IonContent>
+          {
+            this.state.results.length > 0 && (
+              <IonList>
+                {
+                  this.state.results.map((result: any) => (
+                    <IonItem detail={true} key={result.id} routerLink={"/guides/" + result.id}>
+                      <IonThumbnail slot="start">
+                        <img src={"../../assets/images/" + result.id + ".svg"} onError={(e) => {
+                          e.currentTarget.src = "../../assets/shapes.svg"
+                        }} alt={result.title}/>
+                      </IonThumbnail>
+                      <IonLabel>
+                        <h3 className="ion-text-wrap">Read Section Guide</h3>
+                      </IonLabel>
+                    </IonItem>
+                  ))
+                }
+              </IonList>
+            )
+          }
           <div className="ion-padding">
             <div className="akoma-ntoso" ref={this.rootRef}></div>
           </div>
+          <IonToolbar color="light">
```

Comment:
Please rather use the same list mechanism that is used elsewhere, so it looks consistent. Eg see here
https://github.com/laws-africa/constitution-app-react/blob/master/src/pages/cases/Case.tsx#L94